src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -173,12 +177,12 @@
   <Target Name="CopySupplementalTestData"
           AfterTargets="CopyTestToTestDirectory"
           Inputs="@(SupplementalTestData)"
-          Outputs="%(TestTargetFramework.Folder)">
+          Outputs="%(TestNugetTargetMoniker.Folder)">
```

Comment:
Could you make sure that the xunit.performance.runner.Windows packages also get packaged with the supplemental data